monitor/grafana/zeebe.json

```diff
@@ -6960,6 +6960,14 @@
           },
           "pluginVersion": "11.4.0",
           "targets": [
+            {
+              "datasource": {
+                "uid": "$DS_PROMETHEUS"
+              },
+              "expr": "(kubelet_volume_stats_used_bytes{cluster=~\"$cluster\", namespace=~\"$namespace\", persistentvolumeclaim=~\".*camunda.*\"} / kubelet_volume_stats_capacity_bytes{cluster=~\"$cluster\", namespace=~\"$namespace\", persistentvolumeclaim=~\".*camunda.*\"})",
+              "legendFormat": "{{persistentvolumeclaim}}",
+              "refId": "A"
```

Comment:
🔧 I think we dont need to add an extra query, we could also adjust the `persistentvolumeclaim=~".*zeebe.*|.*camunda.*"}`

Reply:
Good point, much better